load.c

```diff
@@ -689,6 +689,19 @@ rb_provide(const char *feature)
 
 NORETURN(static void load_failed(VALUE));
 
+static inline VALUE
+rb_realpath_internal_cached(VALUE hash, VALUE path)
```

Comment:
Generally you should only use the `rb_` prefix for exported functions.  Static functions are not generally `rb_` prefixed.

Reply:
Thanks for the super-fast review, removing `rb_` prefix.